packages/wallets/src/components/DerivAppsSection/DerivAppsGetAccount.tsx

```diff
@@ -97,28 +97,26 @@ const DerivAppsGetAccount: React.FC = () => {
     }, [addTradingAccountToLocalStorage, isAccountCreationSuccess]);
 
     return (
-        <div className='wallets-deriv-apps-section wallets-deriv-apps-section__get-account'>
-            <div className='wallets-deriv-apps-section__icon'>
-                <WalletMarketIcon icon='standard' size='lg' />
-            </div>
-            <div className='wallets-deriv-apps-section__get-content'>
-                <div className='wallets-deriv-apps-section__details'>
-                    <Text size='sm' weight='bold'>
-                        <Localize i18n_default_text='Options' />
-                    </Text>
-                    <Text size={isDesktop ? '2xs' : 'xs'}>
-                        <Localize i18n_default_text='One options account for all platforms.' />
-                    </Text>
-                </div>
+        <TradingAccountCard className='wallets-deriv-apps-section wallets-deriv-apps-section__border'>
```

Comment:
@aizad-deriv, why not put these `classNames` inside the component itself? 🤔  
```suggestion
        <TradingAccountCard className='wallets-deriv-apps-section wallets-deriv-apps-section__border'>
```

Reply:
@jim-deriv This is very specific with Deriv App Section the layout is the same however the behaviour is slightly different from what we have I do agree on one thing where we can add a prop to show a border around it.